include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -17,6 +17,7 @@
 #define dealii_mg_transfer_global_coarsening_h
 
 #include <deal.II/base/mg_level_object.h>
+#include <deal.II/base/mpi_remote_point_evaluation.h>
```

Comment:
I assume this is not needed here, but rather in a follow-up PR? I am ok with putting it here, but I would prefer, if possible, to keep this to the `.templates.h` file and use a forward declaration as necessary.